inst/pages/taxonomy.qmd

```diff
@@ -97,19 +97,18 @@ string with the following scheme `level:value`. If all levels are the same,
 this part is omitted, but can be added by setting `with.rank = TRUE`.
 
 ```{r}
-phylum <- !is.na(rowData(tse)$Phylum) &
-    vapply(data.frame(apply(
-       rowData(tse)[, taxonomyRanks(tse)[3:7]], 1L, is.na)), all, logical(1))
-getTaxonomyLabels(tse[phylum,]) |> head()
-getTaxonomyLabels(tse[phylum,], with.rank = TRUE) |> head()
+# Select those features that have Species level info
+rank_found <- !taxonomyRankEmpty(tse, rank = "Species")
+getTaxonomyLabels(tse[rank_found,]) |> head()
+getTaxonomyLabels(tse[rank_found,], with.rank = TRUE) |> head()
 ```
```

Comment:
For chunks that you modify, please add label in quarto format. Some chunks do not include these labels as we did not follow good practices this strictly. Also, some chunk options are in Rmarkdown format, so these should be converted into quarto format. So this chunk could be something like:

`# | label: get_taxonomy_labels`